clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -1449,6 +1449,39 @@ default DescribeFeaturesResult describeFeatures() {
      */
     UpdateFeaturesResult updateFeatures(Map<String, FeatureUpdate> featureUpdates, UpdateFeaturesOptions options);
 
+    /**
+     * Unregisters a broker.
```

Comment:
We should document the fact that this will only work for self-managed kafka clusters.

Reply:
Ack.